builds/fabric8-che/assembly/dashboard/src/main/patches/src/app/index.module.ts.patch

```diff
@@ -38,12 +43,12 @@
 +    
 +  }
 +
-+  promise.then((resp) => {
-+    console.log('resp : ' + resp.toString() );
-+    if (resp['che.keycloak.disabled'] == 'true') {
-+      keycloakInit(true);
++  promise.then((keycloakSettings) => {
++    console.log('keycloakSettings : ' + keycloakSettings.toString() );
++    if (keycloakSettings['che.keycloak.disabled'] == 'true') {
++      keycloakInit(true, keycloakSettings);
 +    } else {
-+      keycloakInit(false);
++      keycloakInit(false, keycloakSettings);
 +    }
 +  }, (error) => {
 +    console.log(error);
```

Comment:
maybe add for what the error is 
console.log('Error when initializing keycloak', error);